src/fsharp/tast.fs

```diff
@@ -35,12 +35,12 @@ open Microsoft.FSharp.Core.CompilerServices
 /// Unique name generator for stamps attached to lambdas and object expressions
 type Unique = int64
 //++GLOBAL MUTABLE STATE
-let newUnique = let mutable i = 0L in fun () -> System.Threading.Interlocked.Increment(&i)
+let newUnique = let i = ref 0L in fun () -> System.Threading.Interlocked.Increment(i)
```

Comment:
Thanks, my thoughts exactly. Not too experienced with the build system yet, so I'd rather modify code than build parameters.